packages/victory-box-plot/src/index.d.ts

```diff
@@ -14,13 +14,20 @@ import {
   EventPropTypeInterface,
   DomainPropType,
   DomainPaddingPropType,
+  OrientationTypes,
   StringOrNumberOrCallback,
   VictoryDatableProps,
   VictoryCommonProps,
   VictoryStyleInterface,
   VictoryStyleObject
 } from "victory-core";
 
+export type VictoryBoxPlotLabelType =
+  | boolean
+  | any[]
```

Comment:
Wasn't sure about this... should we be enforcing strings and numbers rather than any?

Reply:
I think `victory-core` exposes a `StringOrNumberOrCallback` type that would be useful here

Reply:
Hmm, actually I guess it's a bit different for `VictoryBoxPlot`, but you might be able to borrow some of the ideas from https://github.com/FormidableLabs/victory/blob/master/packages/victory-core/src/index.d.ts#L109

Reply:
And yes, let's use `string | number` instead of any